WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -198,7 +198,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 	}
 
 	/// <summary>
-	/// Mark all the outputs we had in any of our wallets used.
+	/// Mark all the registered outputs we had in any of our wallets used.
```

Comment:
This function doesn't know if the outputs being fed are registered or not.
```suggestion
	/// Mark all the outputs we had in any of our wallets used.
```